guidelines/outline.md

```diff
@@ -0,0 +1,123 @@
+
+	Include other relevant introductory information here, such as a community webpage or contact information.
+
+## Formality and Tone
+
+	Who is the target user for this project and what is their background?
+
+	How would a target user for this project expect to interact with this project?
+
+	Is formal language appropriate for all of your language's Mozilla l10n projects, or only some of them? Which ones?
+
+	Is informal language appropriate for all of your language's Mozilla l10n projects, or only some of them? Which ones?
+
+## Natural expression
+
+	What kind of expression sounds natural in your locale?
+
+	Does your locale allow for many borrowed words and phrases or few?
+
+## Handling cultural references, idioms, and slang
+
+	How are cultural references, idioms, and slang handled in your locale?
+
+	Are there resources for cultural references, idioms, and slang in your language?
+
+## Units and Unit Conversion
+
+	Does your locale use the metric system or the imperial system?
+
+### Date Format
+
+	How are the date formats for weeks and months expressed? Erase the comment markers around the example template or show the format with your own example.
+
+
+    | Name | Format | Example |
+    | ---- | ------ | ------- |
+    | Short | mm/dd/yy | 12/31/99 |
+    | Abbreviated | mmm dd | Dec 31 |
+    | Long | Month Date, Year | December 31, 1999 |
+
+### Calendar view
+
+	Which date is considered the first day of the week, Sunday or Monday?
+
+	Is Lunar calendar observed?  Other regional calendar observed?
+
+### Time Format
+
+	How is time expressed in your language?
+
+### Numerals
+
+	How are numerals and percentages expressed in your language? Erase the comment markers around the example template or show the format with your own example.
+
+| Separator | Character Name | Symbol | Example |
+| --------- | -------------- | ------ | ------- |
+| Decimal | Period | `.` | 1.23 |
+| Thousands | Comma | `,` | 1,234 |
+
+### Currency
+
+	What currency and symbols are used in your country/language for paid apps?
+
+### Units
+
+	Do you use the imperial, metric or nautical system for measuring weight, distance, etc.?
+
+### Address and Postal Code Format
+
+	What is the address format in your language? Erase the comment markers around one of the examples or show the format with your own example.
+
+	Example: Most Asian countries start from big to small:
+
+    [Country]
+    [postal code][state/province][city][district]
+    [street number and name][building and suite numbers]
+    [addressee]
+
+	Countries of European languages start from small to big:
+
+    [addressee]
+    [street number and name][building and suite numbers]
+    [district][city][state/province][postal code]
+    [Country]
+
+### Telephone Number format
+
+	How are telephone numbers expressed in your language? Erase the comment markers around the example template or show the format with your own example.
+
+`(###) ###-####`
```

Comment:
Should we add country code in addition to local code?

Also because of wide use of mobile numbers, some country has different format to distinguish landline and mobile phone. If there is no usage in our case, we can skip this.

Reply:
The outline shouldn't, for the en-US I have no clue how American write phone numbers :)